### PR TITLE
Add rcl_interfaces to standard installation

### PR DESCRIPTION
**Pull request by tfoote**
Connects to ros2/ros2#10

Blocked on ros2/rmw_connext#17

### CODE REVIEW

File: ros2.repos [hunk 1]
```diff
@@ -51,7 +55,7 @@ repositories:
     type: git
     url: https://github.com/ros2/rmw_implementation.git
     version: master
-  ros2/rmw_opensplice:
+ros2/rmw_opensplice:
```
Comment (wjwwood) [inline ros2.repos]: Why did the indention change here?

**Comment by esteve** [comment]
+1

**Comment by wjwwood** [comment]
+1 after comment

File: ros2.repos [hunk 2]
```diff
@@ -31,6 +31,10 @@ repositories:
     type: git
     url: https://github.com/ros2/rcl.git
     version: master
+  ros2/rcl_interfaces:
+    type: git
+    url: git@github.com:ros2/rcl_interfaces.git
```
Comment (dirk-thomas) [inline ros2.repos]: Don't use a git@github url.

**Comment by dirk-thomas** [comment]
+1 after comment

**Pull request merged**